helpers/LSF/Choices.ts

```diff
@@ -1,19 +1,17 @@
-import { LabelStudio } from './LabelStudio';
-
 class ChoicesHelper {
   private get _baseRootSelector() {
-    return '.lsf-choices';
+    return ".lsf-choices";
```

Comment:
will these lsf prefixes need to be updated once we move to monorepo?

Reply:
no, prefixes must remain the same